### PR TITLE
Fix/vertical vid

### PR DESCRIPTION
**Pull request by jesseng11**

### CODE REVIEW

File: src/app/components/jamjarPlayer/jamjarPlayer.html [hunk 1]
```diff
@@ -1,58 +1,50 @@
 <div layout="column" class="theatre-bg">
-  <div layout="column" ng-show="player.jamjar.nowPlaying"
-      class='jamjar-player videogular-container'
```
Comment (drewbanin) [inline src/app/components/jamjarPlayer/jamjarPlayer.html]: @jesseng11 this was the source of the broken bars!

File: src/app/components/jamjarPlayer/jamjarPlayer.html [hunk 2]
```diff
@@ -1,58 +1,50 @@
...
+                vg-theme="player.config.theme"
+                vg-auto-play="(video == player.jamjar.nowPlaying)"
+                vg-player-ready="player.jamjar.onPlayerReady($API, video)"
+                vg-can-play="player.jamjar.onPlayerCanPlay(video)"
+                vg-update-time="player.jamjar.onUpdateTime($currentTime, $duration, video)"
+                vg-change-source="player.jamjar.onUpdateSource($source, video)"
+                vg-update-state="player.jamjar.onUpdateState($state, video)"
+                vg-complete="player.jamjar.onComplete(video)"
+                ng-style="{display: (player.jamjar.nowPlaying == video) ? 'block' : 'none'}"
+                ng-class="{'now-playing-video': (player.jamjar.nowPlaying == video)}">
```
Comment (drewbanin) [inline src/app/components/jamjarPlayer/jamjarPlayer.html]: `.now-playing-video` is only on a videogular element when that video is playing on the screen

**Comment by drewbanin** [comment]
can we also change the overlay timeout to 1.5 seconds so @markkohdev can stop complaining about it? :P

**Comment by jesseng11** [comment]
kay thanks! and i'll change it to 1 sec, since it takes some time to fade out too. merginnn

**Pull request merged**